build.assets/Dockerfile

```diff
@@ -177,6 +177,14 @@ RUN go install github.com/daixiang0/gci@v0.8.2
 RUN (mkdir -p helm-tarball && curl -L https://get.helm.sh/helm-v3.5.2-$(go env GOOS)-$(go env GOARCH).tar.gz | tar -C helm-tarball -xz && \
      cp helm-tarball/$(go env GOOS)-$(go env GOARCH)/helm /bin/ && \
      rm -r helm-tarball*)
+
+# TODO(hugoShaka): remove this backward compatible hack with teleportv13 buildbox
+RUN helm plugin install https://github.com/quintush/helm-unittest && \
```

Comment:
Shouldn't we set `HELM_PLUGINS` for the helm plugin install command, otherwise it would install it in the same location as the old plugin below, no?

Reply:
I tried to do this, but helm did not respect the variable during installation 🙄 this is why the plugin is copied somewhere else, then deleted with `helm plugin uninstall unittest`. In the end we have 3 plugin installed:

- new version in /home/ci/.local/share/helm/plugins-new
- old version in /home/ci/.local/share/helm/plugins
- old version in /root/.local/share/helm/plugins-new